sdk/storage/storage-blob-changefeed/src/utils/utils.common.ts

```diff
@@ -67,12 +67,12 @@ export async function getYearsPaths(
   containerClient: ContainerClient,
   options: GetYearsPathsOptions = {}
 ): Promise<number[]> {
-  const { span, spanOptions } = createSpan("getYearsPaths", options.tracingOptions);
+  const { span, updatedOptions } = createSpan("getYearsPaths", options);
   try {
     const years: number[] = [];
     for await (const item of containerClient.listBlobsByHierarchy("/", {
       abortSignal: options.abortSignal,
-      tracingOptions: { ...options.tracingOptions, spanOptions },
+      tracingOptions: updatedOptions.tracingOptions,
```

Comment:
## Storage Technique #2/3: pass in _only_ tracingOptions

There were a few spots that seemed to be more selective about which fields were copied into the resulting options. For those I just made sure to include the entirety of `tracingOptions` but not do any `...` copying, in case that would be problematic.

Reply:
It's fine for current definition.